e2e-tests/fixtures/Sequence.ts

```diff
@@ -42,42 +47,60 @@ export class Sequence {
     await this.page.getByRole('button', { name: 'Close' }).click();
```

Comment:
I know this isn't necessarily related to this PR, but I think this test can potentially be a little flaky because the page navigates to the `/edit` route after clicking the `Save` button.

Can we add a `waitForURL` in between clicking `Save` and `Close`, just to make sure that the page has navigated to a page with `Close` first?